Runtime/Scripts/MobiledgeXIntegrationConfig.cs

```diff
@@ -74,5 +79,50 @@ public void SetFallbackLocation(double longitude, double latitude)
             fallbackLocation.Longitude = longitude;
             fallbackLocation.Latitude = latitude;
         }
+
+        public static async Task<LocationFromIPAddress> GetLocationFromIP()
+        {
+            HttpClient httpClient = new HttpClient();
+            try
+            {
+                HttpResponseMessage response = await httpClient.GetAsync("https://freegeoip.app/json/").ConfigureAwait(false);
+                string responseBodyStr = response.Content.ReadAsStringAsync().Result;
+                LocationFromIPAddress location = Messaging<LocationFromIPAddress>.Deserialize(responseBodyStr);
+                return location;
+            }
+            catch (Exception)
```

Comment:
can we log the exception?

Reply:
Why ?

Reply:
Might be useful when testing. But also not that important